semmc/semmc.cabal

```diff
@@ -92,7 +92,8 @@ library
                        crucible-llvm,
                        what4-serialize,
                        ordered-containers,
-                       ansi-wl-pprint
+                       ansi-wl-pprint,
+                       bv-sized
```

Comment:
Version bounds for at least this occurrence of bv-sized?

Reply:
Yep, should be fixed in the latest commit